compiler/rustc_transmute/src/layout/mod.rs

```diff
@@ -81,6 +92,16 @@ pub mod rustc {
     }
     impl<'tcx> Ref<'tcx> {}
 
+    impl<'tcx> fmt::Display for Ref<'tcx> {
+        fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+            f.write_char('&')?;
```

Comment:
We could, but it's not relevant to the error message this `Display` routine is currently used in, and I don't anticipate it will be relevant to other transmute error messages, since regions do not impact layout.